### PR TITLE
Make `DeclarationModifier` and `TypeAttribute` normal `TokenSpecSet` instead of generating them

### PR DESCRIPTION
**Pull request by ahoppen**
There’s no reason to generate these `TokenSpecSet` anymore.

### CODE REVIEW

**Comment by ahoppen** [comment]
@swift-ci Please test

**Review by kimdv — APPROVED** [review]
Did you run the code generation? 😅

I see two templates removed from the list

**Comment by ahoppen** [comment]
Oh, yeah. Apparently I missed the deletion of the generated files at some point

**Comment by kimdv** [comment]
@swift-ci please test

**Comment by ahoppen** [comment]
@swift-ci Please test Windows

**Comment by ahoppen** [comment]
@swift-ci Please test

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -11,5 +11,5 @@
      url: https://forums.swift.org/tags/c/development/tooling/39/swift-syntax
      about: Ask and answer questions about SwiftSyntax
    - name: SwiftSyntax Documentation
-     url: https://github.com/apple/swift-syntax/tree/main/Documentation
+     url: https://swiftpackageindex.com/apple/swift-syntax/documentation/swiftsyntax
```
Comment (bnbarham) [inline .github/ISSUE_TEMPLATE/config.yml:14]: Didn't you already change this in another PR?

**Review by bnbarham — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (ahoppen) [reply .github/ISSUE_TEMPLATE/config.yml:14]: Oh, rebase mistake

**Comment by ahoppen** [comment]
@swift-ci Please test

**Comment by ahoppen** [comment]
@swift-ci Please test Windows

**Comment by ahoppen** [comment]
@swift-ci Please test Windows

**Comment by ahoppen** [comment]
@swift-ci Please test

**Comment by ahoppen** [comment]
@swift-ci Please test Windows

**Comment by ahoppen** [comment]
@swift-ci Please test Windows

**Pull request merged**